tests/test_xmlparser.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from pheme.parser.xml import XMLParser
+
+
+@pytest.mark.parametrize("xml", ['<data a="b"/>', '<data><a>b</a></data>',])
+def test_parsing(xml):
+    under_test = XMLParser()
+    result = under_test.parse(xml)
+    assert result == {'data': {'a': 'b'}}
```

Comment:
would be good to get additional tests for the normalization behavior too.

Reply:
The data normalization is done within gvmd.py; there is a different test for that:
tests/test_gvmd_scanreport_datatransformation.py 
the xml parser so far just creates a dict representation of any xml